website/frontend/src/pages/OurProducts/CalibrationPage.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Header from '../../components/OurProducts/Header';
+import Page from '../Page';
+import HeaderImage from '../../assets/img/OurProducts/Calibration/calibration-header.png';
+import { useInitScrollTop } from 'utils/customHooks';
+import Section1Image from '../../assets/img/OurProducts/Calibration/section-1.jpg';
+import Section2Image from '../../assets/img/OurProducts/Calibration/section-2.jpg';
+import Section3Image from '../../assets/img/OurProducts/Calibration/section-3.png';
+import NavTab from '../../components/nav/NavTab';
+import { NextTab } from '../../components/NextTabIcon/NextTab.js';
+
+const CalibrationPage = () => {
+  useInitScrollTop();
+  return (
+    <Page>
+      <div className="product-page calibrate mobile-app analytics api">
+        <Header
+          style = {{ backgroundColor: '#e6e8fa' }}
+          pageTitle={'AirQalibrate'}
+          title={'Calibrate your low-cost sensor data'}
+          subText={
+            'Improve the accuracy and reliability of data from your low-cost air quality monitors using AirQo’s AirQalibrate tool, delivering more accurate results.'
+          }>
+          <img src={HeaderImage} alt="" style={{ borderRadius: '8px'}} />
+        </Header>
+        <div className="content">
+          <div className="grid-wrapper column section section-1">
+            <div className="row">
+              <h2 className="left title">
+                <span className="blue"> Why calibrate</span> your low-cost sensor data?
+              </h2>
+              <p className="right">
+              Low cost sensors are relied on to increase the geographical coverage of air quality monitoring networks. 
+              However, they are sensitive to ambient conditions such as humidity and temperature which could affect 
+              the accuracy of air quality data and require field calibration to improve their performance.              
+              Calibration enhances the quality and reliability of air quality data from low-cost sensors
+              </p>
+            </div>
+            <div className="overlap-section">
+              <div className="lapping-left card" /*card-sm larger-top*/  style={{ backgroundColor: '#FFFCE1' }}>
+                <h5>Cost effective and accessible</h5>
+                <p>
+                AirQalibrate is a Machine Learning based calibration tool that eliminates the need for reference grade monitor 
+                or on-site monitor calibration.<br />
+                <br />
+                AirQalibrate enables users without access to reference grade monitors or technical expertise 
```

Comment:
Change "AirQalibrate enables users" to "It enables users" to avoid redundancy while reading. The name of the tool was already introduced in the paragraph above.